### ISSUE TITLE
Term for -3 SD ≤ OFC < -2 SD

### ISSUE THREAD

**Issue by sansburyf** [obophenotype/human-phenotype-ontology#497]
It is good to keep microcephaly as OFC < -3 SD.  Could we have an HPO term for -3 SD ≤ OFC < -2 SD please though - mainly so that there is a term in Decipher.  Could I also note that the macrocephaly definition does not mirror the microcephaly one (>97th centile = 1.9 SD)?  Thanks.

**Comment by drseb** [comment]
@pnrobinson I suggest to put this as sibling of Microcephaly. Should we do this? What should the class-label then be?

**Comment by markengelstad** [comment]
shouldn't it be a synonym of microcephaly?
as such, would it be usable in DECIPHER?

**Comment by sansburyf** [comment]
There are various definitions of microcephaly and macrocephaly. [For example the fetal alcohol syndrome 4-digit code for microcephaly uses OFC ≤ - 2 SD (≤ 2.3rd centile).  Allanson et al. AJMG 2009;149A:6 in what was supposed to be a consensus statement for dysmorphology used OFC < 3rd centile (< -1.88 SD)]. The more extreme the cut-off, the better the specificity for pathology, so OFC < -3 SD (0.1th centile) , which is the one on HPO, is more likely to indicate abnormality than either of the above examples.

There is of course the issue of what is you use to define normality - and we have an issue in the UK that the UK-WHO curves in use in paediatrics for ages 0-4 (validated for height and weight but not for OFC) differ from the previously used UK-1990 curves (still in use for ages 4-18) by up to 1.4 SD.  In addition, there are the issues of relative vs absolute microcephaly.

The trouble is that a strict definition of microcephaly without qualifiers does not allow you to indicate in a database like Decipher where restricted head growth may be an issue, but because it is ≥ -3 SD, there is an decreased likelihood that it is significant cf. < -3 SD.  Would something like "mild microcephaly (-3 SD ≤ OFC < -2 SD)” work, or is that too wordy?  To make it a synonym of microcephaly would be to broaden the definition.  "Small head size" could be a class label but that sounds subjective.  Allanson et al., though, called "Head circumference, reduced small” and "OFC, small" synonyms of microcephaly in their broad definition, which would include -3 SD ≤ OFC < -2 SD.  You could alternatively have  "mild microcephaly (-3 SD ≤ OFC < 3rd centile)" if you wanted to capture all the people whom Allanson et al. would label with microcephaly.

There is of course the point that whatever decisions are made about microcephaly could reasonably be applied to macrocephaly, but given that macrocephaly has a different definition on HPO, "mild macrocephaly (97th centile < OFC ≤ 3 SD) would have to be a sub-term rather than a parallel term to macrocephaly.

I am copying in Prof. Andrew Jackson whose research includes dwarfism, and with whom I was on an RCPCH (Royal College of Paediatrics and Child Health) working party on OFCs, in case he has any comment to add.

Thanks very much
Francis


Dr Francis H Sansbury,
Locum consultant clinical geneticist,
University Hospitals Bristol NHS Foundation Trust, Clinical Genetics, St. Michael’s Hospital, Southwell Street, Bristol BS2 8EG, tel 0117 34 25653, fax 0117 34 25108.

On 22 Sep 2015, at 17:30, Mark Engelstad <notifications@github.com> wrote:

shouldn't it be a synonym of microcephaly?
as such, would it be usable in DECIPHER?

On Mon, Sep 21, 2015 at 5:55 AM Sebastian Köhler <notifications@github.com>
wrote:

> @pnrobinson <https://github.com/pnrobinson> I suggest to put this as
> sibling of Microcephaly. Should we do this? What should the class-label
> then be?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/obophenotype/human-phenotype-ontology/issues/497#issuecomment-141966345>
> .
>
—
Reply to this email directly or view it on GitHub.
It is good to keep microcephaly as OFC < -3 SD. Could we have an HPO term for -3 SD ≤ OFC < -2 SD please though - mainly so that there is a term in Decipher. Could I also note that the macrocephaly definition does not mirror the microcephaly one (>97th centile = 1.9 SD)? Thanks.




********************************************************************************************************************

This message may contain confidential information. If you are not the intended recipient please inform the
sender that you have received the message in error before deleting it.
Please do not disclose, copy or distribute information in this e-mail or take any action in reliance on its contents:
to do so is strictly prohibited and may be unlawful.

Thank you for your co-operation.

NHSmail is the secure email and directory service available for all NHS staff in England and Scotland
NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and GSi recipients
NHSmail provides an email address for your career in the NHS and can be accessed anywhere

********************************************************************************************************************

**Comment by drseb** [comment]
Would attached solution make sense? I would add the synonyms to the suggested new class then.
![suggestion1](https://cloud.githubusercontent.com/assets/5172897/10042194/3f4e58be-61ea-11e5-823f-90c87446c198.png)

**Comment by sansburyf** [comment]
That looks very reasonable from my perspective.  Thanks very much.

Francis.

On 23 Sep 2015, at 10:58, Sebastian Köhler <notifications@github.com<mailto:notifications@github.com>> wrote:


Warning: This message contains unverified links which may not be safe.  You should only click links if you are sure they are from a trusted source.


Would attached solution make sense? I would add the synonyms to the suggested new class then.
[https://cloud.githubusercontent.com/assets/5172897/10042194/3f4e58be-61ea-11e5-823f-90c87446c198.png]<https://cloud.githubusercontent.com/assets/5172897/10042194/3f4e58be-61ea-11e5-823f-90c87446c198.png>

—
Reply to this email directly or view it on GitHub<https://github.com/obophenotype/human-phenotype-ontology/issues/497#issuecomment-142550052>.


********************************************************************************************************************

This message may contain confidential information. If you are not the intended recipient please inform the
sender that you have received the message in error before deleting it.
Please do not disclose, copy or distribute information in this e-mail or take any action in reliance on its contents:
to do so is strictly prohibited and may be unlawful.

Thank you for your co-operation.

NHSmail is the secure email and directory service available for all NHS staff in England and Scotland
NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and GSi recipients
NHSmail provides an email address for your career in the NHS and can be accessed anywhere

********************************************************************************************************************

**Comment by drseb** [comment]
If @pnrobinson or @markengelstad do not object this solution, I will implement this today.

Thanks,
Sebastian

**Comment by drseb** [comment]
Note that only the class labelled *Mild microcephaly* will reflect your requested definition. Other subclasses of *Microcephaly* (e.g. *Progressive microcephaly*) will follow the definition of OFC < -3SD.

I hope this works for you.

**Comment by sansburyf** [comment]
Thank you; that is very helpful.

Francis.

On 24 Sep 2015, at 13:59, Sebastian Köhler <notifications@github.com<mailto:notifications@github.com>> wrote:


Note that only the class labelled Mild microcephaly will reflect your requested definition. Other subclasses of Microcephaly (e.g. Progressive microcephaly) will follow the definition of OFC < -3SD.

I hope this works for you.

—
Reply to this email directly or view it on GitHub<https://github.com/obophenotype/human-phenotype-ontology/issues/497#issuecomment-142920054>.


********************************************************************************************************************

This message may contain confidential information. If you are not the intended recipient please inform the
sender that you have received the message in error before deleting it.
Please do not disclose, copy or distribute information in this e-mail or take any action in reliance on its contents:
to do so is strictly prohibited and may be unlawful.

Thank you for your co-operation.

NHSmail is the secure email and directory service available for all NHS staff in England and Scotland
NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and GSi recipients
NHSmail provides an email address for your career in the NHS and can be accessed anywhere

********************************************************************************************************************

**Comment by pnrobinson** [comment]
Thanks Francis for this suggestion and Sebastian for the implementation!

**Comment by sansburyf** [comment]
Dear Andrew

Thank you very much.  The attached diagram shows what we now have as a result of this suggestion.  Mild microcephaly is sitting outside microcephaly to avoid altering the previous definition of microcephaly.  One could have moderate (-4 SD ≤ OFC < -3 SD), severe (-5 SD ≤ OFC < -4 SD), and extreme (OFC < -5 SD) as subtypes of microcephaly.  I agree this would be useful (although on Decipher, the text labels are not immediately obvious, so people might be subjective rather than objective about what they put into each subtype.

Dear HPO (Peter, Sebastian et al.)

Would we be able to add these subtypes please?  Is it possible to have the definitions in the name of the subtypes please, rather than as text labels (for the Decipher reason above)?

Could we also do the same for the definitions of macrocephaly please, although there, as mentioned below, mild macrocephaly (97th centile < OFC ≤ 3 SD) would be a subtype of macrocephaly due to the different definition.  We could have moderate (3 SD < OFC ≤ 4 SD), severe (4 SD < OFC ≤ 5 SD) and extreme (5 SD < OFC) macrocephaly in the same way though.

Thanks very much
Francis

On 7 Oct 2015, at 19:03, Andrew Jackson <ajackso8@exseed.ed.ac.uk<mailto:ajackso8@exseed.ed.ac.uk>> wrote:

dear Frances

thanks for copying me in.

two thoughts:-

1. I agree, I would  avoid using the UK-who charts for defining micro-or macrocephaly, given the discrepancy from other growth charts worldwide

2. When we were working on the HPO neurology terminology microcephaly was separated into mild, moderate, severe and extreme  on the basis of standard deviation cut-offs (-2, -3, -4 et cetera). Would this be useful to do?

I would certainly encourage some quantification of microcephaly in terms of severity, as this is important in discriminating different diagnoses.

Very best

Andrew



On 23/09/2015 04:06, Sansbury Francis (UNIVERSITY HOSPITALS BRISTOL NHS FOUNDATION TRUST) wrote:
There are various definitions of microcephaly and macrocephaly. [For example the fetal alcohol syndrome 4-digit code for microcephaly uses OFC ≤ - 2 SD (≤ 2.3rd centile).  Allanson et al. AJMG 2009;149A:6 in what was supposed to be a consensus statement for dysmorphology used OFC < 3rd centile (< -1.88 SD)]. The more extreme the cut-off, the better the specificity for pathology, so OFC < -3 SD (0.1th centile) , which is the one on HPO, is more likely to indicate abnormality than either of the above examples.

There is of course the issue of what is you use to define normality - and we have an issue in the UK that the UK-WHO curves in use in paediatrics for ages 0-4 (validated for height and weight but not for OFC) differ from the previously used UK-1990 curves (still in use for ages 4-18) by up to 1.4 SD.  In addition, there are the issues of relative vs absolute microcephaly.

The trouble is that a strict definition of microcephaly without qualifiers does not allow you to indicate in a database like Decipher where restricted head growth may be an issue, but because it is ≥ -3 SD, there is an decreased likelihood that it is significant cf. < -3 SD.  Would something like "mild microcephaly (-3 SD ≤ OFC < -2 SD)” work, or is that too wordy?  To make it a synonym of microcephaly would be to broaden the definition.  "Small head size" could be a class label but that sounds subjective.  Allanson et al., though, called "Head circumference, reduced small” and "OFC, small" synonyms of microcephaly in their broad definition, which would include -3 SD ≤ OFC < -2 SD.  You could alternatively have  "mild microcephaly (-3 SD ≤ OFC < 3rd centile)" if you wanted to capture all the people whom Allanson et al. would label with microcephaly.

There is of course the point that whatever decisions are made about microcephaly could reasonably be applied to macrocephaly, but given that macrocephaly has a different definition on HPO, "mild macrocephaly (97th centile < OFC ≤ 3 SD) would have to be a sub-term rather than a parallel term to macrocephaly.

I am copying in Prof. Andrew Jackson whose research includes dwarfism, and with whom I was on an RCPCH (Royal College of Paediatrics and Child Health) working party on OFCs, in case he has any comment to add.

Thanks very much
Francis


Dr Francis H Sansbury,
Locum consultant clinical geneticist,
University Hospitals Bristol NHS Foundation Trust, Clinical Genetics, St. Michael’s Hospital, Southwell Street, Bristol BS2 8EG, tel 0117 34 25653, fax 0117 34 25108.

On 22 Sep 2015, at 17:30, Mark Engelstad
<notifications@github.com<mailto:notifications@github.com>>
 wrote:

shouldn't it be a synonym of microcephaly?
as such, would it be usable in DECIPHER?

On Mon, Sep 21, 2015 at 5:55 AM Sebastian Köhler
<notifications@github.com<mailto:notifications@github.com>>

wrote:


@pnrobinson <https://github.com/pnrobinson>
 I suggest to put this as
sibling of Microcephaly. Should we do this? What should the class-label
then be?

—
Reply to this email directly or view it on GitHub

<https://github.com/obophenotype/human-phenotype-ontology/issues/497#issuecomment-141966345>

.


—
Reply to this email directly or view it on GitHub.
It is good to keep microcephaly as OFC < -3 SD. Could we have an HPO term for -3 SD ≤ OFC < -2 SD please though - mainly so that there is a term in Decipher. Could I also note that the macrocephaly definition does not mirror the microcephaly one (>97th centile = 1.9 SD)? Thanks.




********************************************************************************************************************

This message may contain confidential information. If you are not the intended recipient please inform the
sender that you have received the message in error before deleting it.
Please do not disclose, copy or distribute information in this e-mail or take any action in reliance on its contents:
to do so is strictly prohibited and may be unlawful.

Thank you for your co-operation.

NHSmail is the secure email and directory service available for all NHS staff in England and Scotland
NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and GSi recipients
NHSmail provides an email address for your career in the NHS and can be accessed anywhere

********************************************************************************************************************


--

Andrew Jackson FRSE FMedSci
Professor of Human Genetics
MRC Human Genetics Unit
Institute of Genetics and Molecular Medicine
University of Edinburgh
Crewe Road
Edinburgh EH4 2XU
UK
Tel: + 44 (0) 131 651 8500
Fax: + 44 (0) 131 467 8456

e:
andrew.jackson@igmm.ed.ac.uk<mailto:andrew.jackson@igmm.ed.ac.uk>

w:
www.ajackson.hgu.mrc.ac.uk
<Edinburgh University charitable status.txt>
[cid:9FB85369-1005-4F47-BD0F-3E6FE78693AC@home]

********************************************************************************************************************

This message may contain confidential information. If you are not the intended recipient please inform the
sender that you have received the message in error before deleting it.
Please do not disclose, copy or distribute information in this e-mail or take any action in reliance on its contents:
to do so is strictly prohibited and may be unlawful.

Thank you for your co-operation.

NHSmail is the secure email and directory service available for all NHS staff in England and Scotland
NHSmail is approved for exchanging patient data and other sensitive information with NHSmail and GSi recipients
NHSmail provides an email address for your career in the NHS and can be accessed anywhere

********************************************************************************************************************
